src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -41,50 +43,87 @@ const SelectPanelContext = React.createContext<{
   selectionVariant: 'multiple',
 })
 
-// @ts-ignore todo
-const SelectPanel = props => {
-  const anchorRef = useProvidedRefOrCreate(props.anchorRef)
+export type SelectPanelProps = {
+  title: string
+  description?: string
+  selectionVariant?: ActionListProps['selectionVariant'] | 'instant'
+  id?: string
+
+  defaultOpen?: boolean
+  open?: boolean
+  anchorRef?: React.RefObject<HTMLButtonElement>
+
+  onCancel?: () => void
+  onClearSelection?: undefined | (() => void)
+  onSubmit?: (event?: React.FormEvent<HTMLFormElement>) => void
+
+  // TODO: move these to SelectPanel.Overlay or overlayProps
+  width?: AnchoredOverlayProps['width']
+  height?: AnchoredOverlayProps['height']
+
+  children: React.ReactNode
+}
+
+const Panel: React.FC<SelectPanelProps> = ({
```

Comment:
Super random question, is there a convention that is used to decide between `React.FC<Props>` versus annotating the props directly e.g. `const Component = (props: Props) => ...`?

Reply:
Not yet, I don't have strong opinions on this, happy to go either way.

Do we want to avoid `React.FC`?